feature-libs/organization/account-summary/core/account-summary-page-meta.resolver.ts

```diff
@@ -0,0 +1,91 @@
+import { Injectable } from '@angular/core';
+import {
+  BreadcrumbMeta,
+  ContentPageMetaResolver,
+  Page,
+  RoutingService,
+  SemanticPathService,
+  TranslationService,
+} from '@spartacus/core';
+import { OrganizationPageMetaResolver } from '@spartacus/organization/administration/core';
+import { combineLatest, defer, Observable } from 'rxjs';
+import {
+  distinctUntilChanged,
+  map,
+  shareReplay,
+  switchMap,
+} from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AccountSummaryPageMetaResolver extends OrganizationPageMetaResolver {
```

Comment:
this resolver is in `core, but the provider is in component module. 
```
{
      provide: PageMetaResolver,
      useExisting: AccountSummaryPageMetaResolver,
      multi: true,
    },
```
Please move the provider to core module.